### DISCUSSION TITLE
Info HA

### DISCUSSION THREAD

**Discussion by hantrax** [indomus/forum#610] [HUB: Home Assistant]
Ciao ragazzi, ho montato un display oled con Interfaccia I2C sul Raspberry, ho HA installato come Hassio..credo, ho la voce supervisor...
Sto seguendo questa guida **LINK RIMOSSO**
ma non riesco ad eseguire i comandi..
prima di tutto il raspi-config per abiltare I2C
poi sudo apt-get install python-smbus mi dice che apt-get non esiste...
Prima avevo installato HA manualmente e riuscivo a fare tutto....
con Hassio è diverso?

**Comment by snakuzzo** [comment]
Non sono ammessi link a siti terzi. Rimuovo.
Leggi il regolamento

**Comment by hantrax** [comment]
scusate...mi era sfuggito....
comunque dovrei eseguire questi comandi per installare il supporto al display e poi uno script per la gestione... 
sudo apt-get install python-smbus
sudo apt-get install i2c-tools
sudo pip3 install Adafruit_BBIO
ma è come se fosse disabilitato qualcosa

**Reply by snakuzzo** [reply]
appena cercato usando apt

```
pi@raspberrypi:~ $ sudo apt-get update
Get:1 http://archive.raspberrypi.org/debian buster InRelease [32.6 kB]
Get:2 https://download.docker.com/linux/raspbian buster InRelease [33.6 kB]                                               
Get:3 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]                     
Get:4 http://archive.raspberrypi.org/debian buster/main armhf Packages [378 kB]
Get:5 https://download.docker.com/linux/raspbian buster/stable armhf Packages [13.8 kB]
Get:6 http://raspbian.raspberrypi.org/raspbian buster/main armhf Packages [13.0 MB]
Fetched 13.5 MB in 11s (1260 kB/s)                                                                                        
Reading package lists... Done
pi@raspberrypi:~ $ 
pi@raspberrypi:~ $ sudo apt-cache search python-smbus
python-smbus - Python bindings for Linux SMBus access through i2c-dev
pi@raspberrypi:~ $ 
pi@raspberrypi:~ $ sudo apt-cache search i2c-tools
i2c-tools - heterogeneous set of I2C tools for Linux
```

Quanto al modulo python...

https://pypi.org/project/Adafruit-BBIO/

Come vedi c'è tutto.

**Reply by snakuzzo** [reply]
P.S. non rispondere aprendo un altro thread, ma nel caso facendo reply sotto l'ultimo messaggio

**Reply by hantrax** [reply]
grazie...e scusa se non sono molto pratico....
sicuramente mi perdo qualcosa...
mi collego via ssh o utilizzando il terminal...ma poi quando scrivo sudo apt-get update mi dice che il comando apt-get non è stato trovato...

login as: pi
pi@192.168.2.200's password:

| |  | |                          /\           (_)   | |            | |
| |__| | ___  _ __ ___   ___     /  \   ___ ___ _ ___| |_ __ _ _ __ | |_
|  __  |/ _ \| '_ \ _ \ / _ \   / /\ \ / __/ __| / __| __/ _\ | '_ \| __|
| |  | | (_) | | | | | |  __/  / ____ \\__ \__ \ \__ \ || (_| | | | | |_
|_|  |_|\___/|_| |_| |_|\___| /_/    \_\___/___/_|___/\__\__,_|_| |_|\__|

Welcome to the Home Assistant command line.

System information
  IPv4 addresses for eth0:  xxxxxx
  IPv6 addresses for eth0:  xxxxxxxx
  IPv4 addresses for wlan0:

  OS Version:               Home Assistant OS 6.2
  Home Assistant Core:      2021.8.6

  Home Assistant URL:       http://homeassistant.local:8123
  Observer URL:             http://homeassistant.local:4357
➜  ~ sudo apt-get update
sudo: apt-get: command not found
➜  ~

**Reply by snakuzzo** [reply]
Ah scusa...hai hassos, m'era sfuggito questo passaggio. Non puoi farlo. Non puoi installare questa roba su HassOS. Poi non so se con ssh & terminal, sbloccando la modalità avanzata, hai a disposizione apt

**Reply by hantrax** [reply]
me lo immaginavo....
grazie comunque!!

**Reply by snakuzzo** [reply]
Provato io per te...non puoi.

**Reply by hantrax** [reply]
ok...
